sparkle/src/icons/src/solid/TSquare.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
OOC: to which SVGs do we allow/disallow passing props? (I don't know much about which props are used for svg besides the color)